give.php

```diff
@@ -326,6 +337,7 @@ private function includes() {
 			require_once GIVE_PLUGIN_DIR . 'includes/actions.php';
 			require_once GIVE_PLUGIN_DIR . 'includes/filters.php';
 			require_once GIVE_PLUGIN_DIR . 'includes/api/class-give-api.php';
+			require_once GIVE_PLUGIN_DIR . 'includes/class-notices.php';

```

Comment:
@mehul0810 I am taking all notices ( frontend and backend ) under one class `Give_Notices`.